src/components/popups/FeaturedPhoto.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import XIcon from '@/components/icons/XIcon.vue';
+
+const prop = defineProps({
+	featuredPhoto: {
+		type: Object,
+		required: true,
+	},
+});
+defineEmits(['close']);
+</script>
+
+<template>
+	<Teleport to="#popup">
+		<div
+			class="popup bg-darkBG dark:bg-gray5 modal-animation fixed top-0 bottom-0 left-0 right-0 z-30 h-screen w-full flex flex-col justify-center items-center pt-16 bg-opacity-50 dark:bg-opacity-50"
+			@click.stop="$emit(`close`)"
+		>
+			<!-- Container -->
+			<button class="bg-gray1 dark:bg-gray5 focus:outline-none mb-5 rounded-full p-1" @click="$emit(`close`)">
```

Comment:
When I try on a little screen, the caption is cut, maybe having a max-height on the photo is necessary?
<img width="1509" alt="Capture d’écran 2022-09-14 à 16 15 18" src="https://user-images.githubusercontent.com/27286861/190179357-c9fdb9f0-eddb-4185-9939-0188ebb032e7.png">